src/domain/Coin.ts

```diff
@@ -0,0 +1,39 @@
+import { COINS } from '../constants';
+import { pickRandomElement } from '../utils';
+
+interface ICoin {
+  500: number;
+  100: number;
+  50: number;
+  10: number;
+}
+
+class Coin implements ICoin {
+  500: number = 0;
+  100: number = 0;
+  50: number = 0;
+  10: number = 0;
```

Comment:
```suggestion
class Coin {
  500 = 0;
  100 = 0;
  50 = 0;
  10 = 0;
```
외부에서 활용할 일 없는 인터페이스라면 implement할 필요없을거같아요.
클래스 내부에서도 초기화를 0이라는 number 타입으로 추론을 해줘서 명시할 필요는 없을거같네요

Reply:
외부 활용 여부에 따라 implement를 결정할 수도 있겠네요!
위 사례의 숫자값 초기화와 같이 타입 추론이 명백한 경우, 리뷰어님께서는 보통 타입을 명시하지 않는 편이신가요?

Reply:
반영 링크입니다! [4b23e71](https://github.com/woowacourse/javascript-vendingmachine/pull/32/commits/4b23e715a101153e2c62cc72dbd01aa553624897)

Reply:
> 외부 활용 여부에 따라 implement를 결정할 수도 있겠네요! 위 사례의 숫자값 초기화와 같이 타입 추론이 명백한 경우, 리뷰어님께서는 보통 타입을 명시하지 않는 편이신가요?

저는 보통 명시하지 않는편입니다~
특히 저렇게 원시값을 직접 할당해 초기화하는 경우에는 더 필요성이 떨어진다고 생각해요!